src/vpn-api/worker.ts

```diff
@@ -22,9 +22,11 @@ import * as forever from 'forever-monitor';
 import * as morgan from 'morgan';
 import * as net from 'net';
 
+import { logger } from '../utils';
+import * as errors from '../utils/errors';
```

Comment:
If you change this (and similar places) back to
```ts
import { captureException, Raven } from './utils/errors';
```
then it'll make for a much smaller diff